src/it/java/com/google/checkstyle/test/base/AbstractGoogleModuleTestSupport.java

```diff
@@ -129,6 +140,30 @@ else if (moduleId == null) {
         return result;
     }
 
+    /**
+     * Returns list of {@link Configuration} instance for the list of module name.
+     * This implementation uses {@link #getModuleConfig(String)} method inside.
+     *
+     * @param moduleName list of module name.
+     * @param moduleId list of module id.
+     * @return list of {@link Configuration} instance for the given module name.
+     * @throws IllegalStateException if there is a problem retrieving the module or config.
+     */
+    protected static List<Configuration>
+        getModuleConfig(List<String> moduleName, List<String> moduleId) {
```

Comment:
please name it as `getModuleByIds(List<String> moduleId)`, if we have id, it is enough to identify module.
Searching of filter by "id" property is hack , but I am ok with it for now.

Reply:
@romani done.